### PR TITLE
Ankit | Test | Prod - Plus icon is coming on invoice, bill, credit/debit note in side bar in company mode

### PR DESCRIPTION
**Pull request by ankitdubeyoffice**
https://app.clickup.com/t/86cvbjqjp

### CODE REVIEW

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html [hunk 1]
```diff
@@ -218,7 +218,7 @@
                                                         >
                                                             <i [ngClass]="item.icon"></i>
                                                             {{ item?.label }}
-                                                            <ng-container *ngIf="currentOrganizationType === 'BRANCH'">
+                                                            <ng-container *ngIf="currentOrganizationType === 'BRANCH' || isCompanyWithNoBranch">
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html]:
```suggestion
                                                            <ng-container *ngIf="currentOrganizationType === 'BRANCH' || isCompanyWithoutBranch">
```

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 2]
```diff
@@ -111,6 +111,8 @@ export class PrimarySidebarComponent implements OnInit, OnChanges, OnDestroy {
     /** True if account has unsaved changes */
     public hasUnsavedChanges: boolean = false;
     public commandkDialogRef: MatDialogRef<any>;
+    /** The company has no branch */
+    public isCompanyWithNoBranch: boolean = false;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]:
```suggestion
    public isCompanyWithoutBranch: boolean = false;
```

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 3]
```diff
@@ -111,6 +111,8 @@ export class PrimarySidebarComponent implements OnInit, OnChanges, OnDestroy {
     /** True if account has unsaved changes */
     public hasUnsavedChanges: boolean = false;
     public commandkDialogRef: MatDialogRef<any>;
+    /** The company has no branch */
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]:
```suggestion
    /** Holds true if company has no branch */
```

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts [hunk 4]
```diff
@@ -255,6 +257,7 @@ export class PrimarySidebarComponent implements OnInit, OnChanges, OnDestroy {
         });
         this.currentCompanyBranches$.subscribe(response => {
             if (response && response.length) {
+                this.isCompanyWithNoBranch = response?.length === 1;
```
Comment (dvCodeWorld) [inline apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]:
```suggestion
                this. isCompanyWithoutBranch = response?.length === 1;
```

**Review by dvCodeWorld — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts (hunk 4 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]: All done

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts (hunk 3 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]: done

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts (hunk 2 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.ts]: done

File: apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html (hunk 1 above)
Comment (ankitdubeyoffice) [reply apps/web-giddh/src/app/shared/primary-sidebar/primary-sidebar.component.html]: done

**Review by dvCodeWorld — APPROVED** [review]

**Pull request merged**